src/app/components/content/IsaacCardDeck.tsx

```diff
@@ -9,7 +10,15 @@ interface IsaacCardDeckProps {
 
 export const IsaacCardDeck = ({doc, className}: IsaacCardDeckProps) => {
     const classNameString: string = className !== undefined ? className : "";
-    return <div className={"card-deck isaac-cards-body " + classNameString}>
-        {doc?.cards?.map((props, i) => <IsaacCard key={i} doc={props}/>)}
-    </div>
+    return <Container>
+        {doc.title && <Row className="my-4">
+            <Col>
+                <h3 className="h-title text-center">{doc.title}</h3>
+            </Col>
+        </Row>}
+        <Row className={"card-deck isaac-cards-body my-3" + classNameString}>
```

Comment:
```suggestion
        <CardDeck className={"card-deck isaac-cards-body my-3" + classNameString}>
```
Changing this `Row` to a `CardDeck` makes the cards stack nicely on small screen sizes. Currently both on live and here the cards look quite misaligned in this case.

Reply:
Best place to observe this is `/alevel`

Reply:
Agreed, that looks much better.